### ISSUE TITLE
[proposal] test coverage output

### ISSUE THREAD

**Issue by RossComputerGuy** [ziglang/zig#18860]
It would be useful to have like a dot graphs generation or HTML output generation of a module's test coverage. Maybe it could show which lines have test failures along with the error? A lot of test frameworks have this ability and so having this could make it easier to determine what has been tested and what has not been tested.

**Comment by squeek502** [comment]
If you haven't seen it, here's a way to get coverage with status-quo Zig: https://zig.news/squeek502/code-coverage-for-zig-1dk1

Somewhat related (since built-in coverage would depend on instrumentation): https://github.com/ziglang/zig/issues/5484

**Comment by RossComputerGuy** [comment]
@squeek502 Yeah those do look nice but I believe something in Zig itself would be better imo. Those look like they rely on debug info and the symbol table. Using the actual Zig AST and/or the test runner could give better test coverage output.

**Comment by VisenDev** [comment]
Possibly related

https://github.com/ziglang/zig/issues/17821

**Comment by squeek502** [comment]
Duplicate of #352
